pkg/hubble/observer/local_observer.go

```diff
@@ -370,12 +399,21 @@ nextEvent:
 			// when a query asks for 20 events, then lost events should not be
 			// accounted for as they are not events per se but an indication
 			// that some event was lost).
-			resp = &observerpb.GetFlowsResponse{
-				Time:     e.Timestamp,
-				NodeName: nodeTypes.GetAbsoluteNodeName(),
-				ResponseTypes: &observerpb.GetFlowsResponse_LostEvents{
-					LostEvents: ev,
-				},
+
+			// We only want to rate-limit lost events that originate from the
+			// ring buffer. Other lost events should be rate-limited closer to
```

Comment:
nit: clarification about which ringbuf to avoid ambiguity :sweat_smile: 

```suggestion
			// Hubble ring buffer. Other lost events should be rate-limited closer to
```